### DISCUSSION TITLE
Request support for RFC7871

### DISCUSSION THREAD

**Discussion by liang-hiwin** [semihalev/sdns#248] [Ideas]
Request support for RFC7871, https://datatracker.ietf.org/doc/html/rfc7871

**Comment by semihalev** [comment]
Before I implemented that but after I deleted the code because of privacy concerns. This can be add again with on/off config option.

**Reply by liang-hiwin** [reply]
ok

**Comment by semihalev** [comment]
Implemented and will be release on v1.7.0
